include/RAJA/util/View.hpp

```diff
@@ -68,7 +68,10 @@ struct View {
   {
   }
 
-  RAJA_INLINE constexpr View(View const &) = default;
```

Comment:
Do you have any insight as to way this changes is required?

Reply:
Is this something new showing up with CUDA 9.2? As we discussed yesterday, I've been using the OffsetLayout+View combo in SAMRAI with CUDA 8.0 without issue.

Reply:
It came up with cuda/9.0.176. I believe we tried CUDA 8 and reproduced the bug as well.  I am not sure about CUDA 9.2, I can try that version.

Reply:
Yup threw run time errors with cuda 9.2 as well.

Reply:
No it isn't @davidbeckingsale, it's a relatively rare issue that @ajkunen and I have been fighting in the guts of the nested loop stuff for a while.  It doesn't always happen, but there's a reason so many of those internal functors and things have explicit copy constructors.  Sometimes nvcc just plain fails to emit a correct default copy constructor for the device, it seems to happen mostly (but not only) with things that inherit from something that also needs copy construction, and only on things that are captured, but what the actual cause is completely eludes me.  This may actually be the smallest, easiest to reproduce version of the bug I've seen though, so it would be really good to use this as a bug report case.

Reply:
Just so we have it here, I think [this commit](https://github.com/LLNL/RAJA/commit/1e087a620720a431cd8a014fa0a8b0377d123720) was the response to the first time that cropped up last year.

Reply:
I made a different PR  (#478 ) for NVIDIA. The bug doesn't seem to occur with the latest nvcc compiler just the previous ones.

Reply:
so what was the issue?  this PR just says that it's a bug fix.  we really need to document what it's fixing?
was this a compile time error, or runtime error?
Could you add the description to the PR description?  
Thanks!

Reply:
We could add a comment to the constructor saying why it's required, and have a link back to this PR.

Reply:
Both of those would probably be good.  It boils down to "the generated copy constructor didn't actually copy-construct the object," but making it clear why this apparently unnecessary constructor exists is probably a good idea.

Reply:
@davidbeckingsale , do you mean add the url for this PR in view.hpp ?

Reply:
Yessir.